docs/integration-guides/swapping/swaps-with-sor-sdk.md

```diff
@@ -11,77 +11,72 @@ _This guide uses the Balancer API SOR which will find the best result using v2 a
 
 ```typescript
 import {
-  BalancerApi,
-  ChainId,
-  Slippage,
-  SwapKind,
-  Token,
-  TokenAmount,
-  Swap,
-  SwapBuildOutputExactIn,
-  SwapBuildCallInput,
-  ExactInQueryOutput
-} from "@balancer/sdk";
-
-// User defined
+    Token,
+    TokenAmount,
+    ChainId,
+    Slippage,
+    SwapKind,
+    SwapBuildCallInput,
+    SwapBuildOutputExactIn
+} from '../../src';
```

Comment:
@mkflow27 I don't think importing from `src` is compatible with the context of this docs page which approaches from perspective of dev who installs `@balancer/sdk` into their project